packages/react-input/etc/react-input.api.md

```diff
@@ -10,10 +10,18 @@ import type { ObjectShorthandProps } from '@fluentui/react-utilities';
+export const Input: React_2.FunctionComponent<InputProps>;
+
+// @public (undocumented)
+export interface InputCommons {
```

Comment:
Nit: InputCommonProps?

Reply:
I was copying the `___Commons` convention from a few of the other converged components, for things shared between props and state. So I'd probably prefer to keep it as-is, but I don't feel super strongly about it.